_posts/2022-09-07-#25957.md

```diff
@@ -10,14 +10,26 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- This PR is a re-attempt of [PR 15845](https://github.com/bitcoin/bitcoin/pull/15845)
+from 2019, which was closed without being merged. PR 15845 was the
+subject of an earlier [review club](https://bitcoincore.reviews/15845).
+- Please see the notes for that review club meeting, as they apply here too.
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+1. Did you review the PR?
+[Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+1. Why would a node operator enable BIP 158 filters (`-blockfilterindex=1`)? Does this PR affect the motivation?
+1. What downsides, if any, are there to enabling BIP 158 filters?
+1. What is the advantage of descriptor wallets compared to legacy wallets, especially w.r.t. the creation of the filter set?
```

Comment:
Less abbreviations is usually more clear, especially for not super common ones (like e.g. or i.e. which I think is fine)
```suggestion
1. What is the advantage of descriptor wallets compared to legacy wallets, especially with regards to the creation of the filter set?
```